db/structure.sql

```diff
@@ -49,8 +49,7 @@ SET search_path = public, pg_catalog;
 
 CREATE TYPE format_enum AS ENUM (
     'html',
-    'markdown',
-    'text'
```

Comment:
If removing the text option from format_enum is deliberate, then I think we need a migration too.